programs/wen_new_standard/src/instructions/mint/create.rs

```diff
@@ -66,15 +59,6 @@ pub struct CreateMintAccount<'info> {
         associated_token::authority = receiver,
     )]
     pub mint_token_account: Box<InterfaceAccount<'info, TokenAccount>>,
-    /// CHECK: This account's data is a buffer of TLV data
-    #[account(
-        init_if_needed,
-        space = get_meta_list_size(None),
-        seeds = [META_LIST_ACCOUNT_SEED, mint.key().as_ref()],
-        bump,
-        payer = payer,
-    )]
-    pub extra_metas_account: UncheckedAccount<'info>,
```

Comment:
we dont need this?

Reply:
Yeah only need it when royalties are added, since transfer hook is not called without royalties enforced